ray-operator/config/samples/ray_v1alpha1_rayjob.shutdown.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: ray.io/v1alpha1
+kind: RayJob
+metadata:
+  name: rayjob-sample
+spec:
+  entrypoint: python /home/ray/samples/sample_code.py
+  # runtimeEnv decoded to '{
+  #    "pip": [
+  #        "requests==2.26.0",
+  #        "pendulum==2.1.2"
+  #    ],
+  #    "env_vars": {
+  #        "counter_name": "test_counter"
+  #    }
+  #}'
+  # ShutdownAfterJobFinishes specifies whether the RayCluster should be deleted after the RayJob finishes. Default is false.
+  shutdownAfterJobFinishes: true
```

Comment:
The only change in the new sample YAML is setting `shutdownAfterJobFinishes` to true.

Reply:
It's not ideal to copy the entire YAML file and only change one line, because if this YAML file needs to be updated then it's hard to remember to update the other one.  Another approach would be to update `shutdownAfterJobFinishes` in the original YAML file at runtime, but it doesn't seem to fit in well with the current test framework (which accepts a list of sample YAML files to test.)